aio/build.sh

```diff
@@ -57,7 +58,7 @@ source $pythonvenv/bin/activate
 
 ## prerequisites in pip packages
 python -m pip install --upgrade pip
-pip install --upgrade pygraphviz pydot pydotplus requests asyncio
+pip install --upgrade pygraphviz pydot pydotplus requests asyncio orjson
```

Comment:
```suggestion
pip install --upgrade asyncio orjson pydot pydotplus pygraphviz requests
```
list the packages in alphabetical order for easier maintenance